core/primitives/src/version.rs

```diff
@@ -22,7 +22,7 @@ pub const DB_VERSION: DbVersion = 30;
 pub use near_primitives_core::types::ProtocolVersion;
 
 /// Oldest supported version by this client.
-pub const OLDEST_BACKWARD_COMPATIBLE_PROTOCOL_VERSION: ProtocolVersion = 34;
+pub const OLDEST_BACKWARD_COMPATIBLE_PROTOCOL_VERSION: ProtocolVersion = 39;
```

Comment:
let's add a comment above on when/how this value  can be changed in the future.

(so that people know that it is changeable - but under some conditions)

Reply:
and just before merging - let's make sure that we notify people from nodeX and indexer (just in case)

Reply:
Ok, I updated the documentation.